detailed/assets/javascripts/app.js

```diff
@@ -2,6 +2,13 @@
 
 var chaiseRecordApp = angular.module("chaiseRecordApp", ['ngResource', 'ngRoute', 'ui.bootstrap','ui.grid', 'ui.grid.resizeColumns', 'ui.grid.pinning', 'ui.grid.selection', 'ui.grid.moveColumns', 'ui.grid.exporter', 'ui.grid.grouping', 'ui.grid.infiniteScroll', 'ngCookies', 'ngSanitize', 'chaise.utils', 'chaise.navbar']);
 
+// Configure all tooltips to be attached to the body by default. To attach a
+// tooltip on the element instead, set the `tooltip-append-to-body` attribute
+// to `false` on the element.
+chaiseRecordApp.config(['$tooltipProvider', function($tooltipProvider) {
+    $tooltipProvider.options({appendToBody: true});
+}]);
+
```

Comment:
Note: RecordSet, RecordEdit, Viewer, and Record apps uses version 1.3.2 of Angular UI, and the tooltip provider's name in that version is `$uibTooltipProvider`. The "Detailed" app uses its own, older, version of Angular UI, and the provider for that version of tooltip is called `$tooltipProvider`, which is why this config block is different from the other apps'.

I decided to continue allowing Detailed app to use its own version of Angular UI because it's an older app we don't develop new features for anymore and I don't want to force this app to upgrade and potentially break features that use other parts of Angular UI.